flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -1738,6 +1765,28 @@ SqlNode SqlRichExplain() :
     }
 }
 
+/**
+* Parses a execute statement.
+*/
+SqlNode SqlExecute() :
+{
+    SqlParserPos startPos;
+    SqlNode stmt;
+}
+{
+    <EXECUTE>{ startPos = getPos(); }
+    (
+        stmt = SqlStatementSet()
+        |
+        stmt = OrderedQueryOrExpr(ExprContext.ACCEPT_QUERY)
```

Comment:
Just to double check: Does this include the `WITH` clause?

https://modern-sql.com/feature/with

Reply:
yes, OrderedQueryOrExpr includes with clause. do you have any concern on this? Following is the example:
execute|explain insert into t1 with t2 as (select * from t3) select * from t2;
execute|explain with t2 as (select * from t3) select * from t2;
execute|explain statement set begin insert into t1 with t2 as (select * from t3) select * from t2;